File: guess_the_num.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main(){
int ci;
int ui;
int gss = 0;

srand(time(0));
ci = rand()%100 + 1;
printf("%d", ci);
while (ui!=ci)
{
    printf("\nguess the number: ");
    scanf("%d", &ui);
    gss++;

    if (ui == ci)               
    {
        printf("You guess it right");
    }
    else if (ci > ui)
    {
        printf("Choose big num");
    }
    else if (ci<ui)
    {
        printf("Choose small num");
    } 

}

printf(" it was %d you guess it in %d times", ci, gss);

return 0;
}

File: guessTheNum.py
import random  

# random number between 1 to 100
rd = random.randint(1, 100)

# user guesses none mean nothing
userGuess = None


guess = 0


while userGuess != rd:

   
    userGuess = int(input("Guess the number: "))

    
    guess += 1

    
    if rd == userGuess:
        print("you guess it write \n")
    elif rd > userGuess:
        print("Choose! large num  ")
    else:
        print("Choose! Small num  ")

print(f"it was {rd} and you guess it in {guess} times")


'''For store  hi score '''

